cmd/macaroon/brave-skus/free_premium_dev.yaml

```diff
@@ -0,0 +1,13 @@
+tokens:
+  - id: "brave-free-premium-trial sku token v1"
+    version: 1
+    location: "premiumfreetrial.bsg.brave.software"
+    first_party_caveats:
+      - sku: "brave-free-premium-trial"
+      - price: 0
+      - currency: "USD"
+      - description: "Free trial access to Brave premium products"
+      - credential_type: "time-limited"
+      - credential_valid_duration: "P1M"
+      - payment_methods: ["stripe"]
+      - stripe_product_id: "price_1IqgVpHof20bphG6wcCByrRf"
```

Comment:
we should investigate whether we can programatically generate stripe products based on SKU information, this won't work for general case